vincentmuriuki/API_3
# #162155613 POST new order

## What does this PR do?
- It adds the `POST` endpoint used for posting orders
## How to test the endpoint
- git clone 'git@github.com:vincentmuriuki/API_3.git'
- Navigate to `API_3` directory
- Create a virtual environment: `virtualenv venv`
- Then activate the virtual environment: 
                  Windows: `source venv/Scripts/activate`
                  Linux: `source/bin/activate`
- Run `pip install -r requirements.txt`
- Then run `python run.py`
- Open `Postman` and add the following url `http://127.0.0.1:5000/api/v1/orders/new`
- Ensure you use the `POST` request then go to the body section of the and add the following:
{
           "name" : "Pizza", 
           "price" : "150", 
           "quantity": "5"
}
Hit `Enter`
You will get a result such as the one below:

## Screenshots
![post](https://user-images.githubusercontent.com/18227536/48918445-a3c9fd00-ee9d-11e8-8169-9d00a05f4270.png)
